packages/ai-lab/src/components/AILabVideo/AILabLocalVideo.tsx

```diff
@@ -186,6 +197,20 @@ export const AILabLocalVideo = ({ perf, perfCallback, src }: VideoProps) => {
     setPerfProps(undefined);
   }
 
+  useEffect(() => {
+    setInterval(
```

Comment:
Don'e forget to clear the interval when the component unmounts or you'll run into memory leaks here... or you can use this react `useInterval` hook :D 

```tsx
// useInterval: a React hook for setting up intervals without the pitfalls of using setInterval.
type IntervalFunction = () => unknown | void;
function useInterval(callback: IntervalFunction, delay: number) {
  const savedCallback = React.useRef<IntervalFunction | null>(null);

  // Remember the latest callback.
  React.useEffect(() => {
    savedCallback.current = callback;
  }, [callback]);

  // Set up the interval.
  React.useEffect(() => {
    function tick() {
      savedCallback.current?.();
    }
    const id = setInterval(tick, delay);
    return () => clearInterval(id);
  }, [delay]);
}
```

usage:

```tsx
function ReactComponent() {

  useInterval(() => {
    // do something here
  }, intervalTimeInMS);

  return ();
}
```

Reply:
👍  I will make sure that I don't leave any intervals running when I go to get this as a real PR. The intention of this was a quick test to allow Gant to take a look at it since I am getting incredibly slow perf on inference on my machine.